reana_job_controller/job_manager.py

```diff
@@ -130,3 +133,10 @@ def cache_job(self):
     def update_job_status(self):
         """Update job status in DB."""
         pass
+
+    def _extend_env_vars(self, env_vars):
```

Comment:
What is the use case for this?

What do you think about returning just the REANA specific variables (then maybe changing the name of the function to something like `_get_reana_env_variables`) and then [when we use it](https://github.com/reanahub/reana-job-controller/pull/178/files#diff-39cea9b4bf76e1e8f673a7287fcc5676R52) we explicitly extend the user-provided env vars with the REANA ones?